video_info.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from datetime import datetime
+
+class VideoInfo:
+    # Where the CLI is accessing the DB, chose to access my local machine
+    def __init__(self, url="http://localhost:5000"):
+        self.url = url
+        self.selected_video = None
```

Comment:
Very cool that you used this attribute to track the selected video!
Very few submissions had this functionality